### PR TITLE
using pre-commit now, removed coala config

### PR DESCRIPTION
**Pull request by goern**
Signed-off-by: Christoph Görn <goern@redhat.com>

Migrating to pre-commit

## This introduces a breaking change

- [X] Yes
- [ ] No

## Description

This is a migration from Coala to pre-commit, some files have been changed by pre-commit too. There is still issues with pydocstyle.

### CODE REVIEW

**Comment by harshad16** [comment]
> Do we need setup.py for this component?

as discussed here, https://github.com/thoth-station/unresolved-package-handler/pull/47#issuecomment-650215895
this component was not considered as a package.

**Comment by pacospace** [comment]
@goern

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+from setuptools.command.test import test as TestCommand
```
Comment (pacospace) [inline setup.py:1]: this should be removed :)

**Comment by pacospace** [comment]
@harshad16 @goern nothing happens

**Pull request merged**